Makefile

```diff
@@ -61,6 +61,21 @@ integration-coverage:
 integration-cover-all: PACKAGES=./...
 integration-cover-all: integration-coverage
 
+.PHONY: fuzz-validate-cover
```

Comment:
nit: for a global namespace like here in the make file, this name is a bit generic. It is unclear what "validate" is fuzzed.

Reply:
there is an issue to parametrize this target (https://github.com/0xsoniclabs/sonic-admin/issues/178) there it should also be renamed to `make fuzz {testname} {fuzztime}`. 
But I will rename it now to `fuzz-txpool-validatetx-cover`